### PR TITLE
Fix cargo-deny failing on webc crate

### PR DESCRIPTION
**Pull request by fschutt**

### CODE REVIEW

**Comment by theduke** [comment]
I guess it fails because of the unmaintained warnings?

Guess we'll need to whitelist those as well.

File: deny.toml [hunk 1]
```diff
@@ -169,6 +169,7 @@ deny = [
 ]
 # Certain crates/versions that will be skipped when doing duplicate detection.
 skip = [
+    { name = "webc" },
```
Comment (theduke) [inline deny.toml:172]: Please add comments on why these are whitelisted.

**Review by theduke — APPROVED** [review]
Approved after the mentioned comments are added.

File: deny.toml (hunk 1 above)
Comment (ptitSeb) [reply deny.toml]: done

**Comment by theduke** [comment]
@fschutt I actually meant comments for all the crates, not just webc, because we will have no idea why they are whitelisted in a months time.

But we can just include that in the #3426 PR.

**Pull request merged**